data/sql/updates/pending_db_world/rev_1662378898044710600.sql

```diff
@@ -0,0 +1,38 @@
+
+DELETE FROM `smart_scripts` WHERE  `entryorguid` IN (36452, 36741) AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES 
+(36452, 0, 0, 1, 19, 0, 100, 0, 14465, 0, 0, 0, 0, 85, 69256, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Gwen Armstead SAI Script'),
+(36452, 0, 1, 2, 61, 0, 100, 0, 0, 0, 0, 0, 0, 85, 82892, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Gwen Armstead SAI Script'),
+(36452, 0, 2, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 85, 105011, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Gwen Armstead SAI Script'),
+(36741, 0, 0, 0, 60, 0, 100, 513, 1000, 1000, 0, 0, 0, 53, 1, 36741, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Swift Mountain Horse SAI Script'),
+(36741, 0, 1, 0, 40, 0, 100, 512, 2, 36741, 0, 0, 0, 9, 0, 0, 0, 0, 0, 0, 14, 9956226, 196399, 0, 0, 0, 0, 0, 'Swift Mountain Horse SAI Script'),
+(36741, 0, 2, 0, 40, 0, 100, 512, 3, 36741, 0, 0, 0, 9, 0, 0, 0, 0, 0, 0, 14, 9956226, 196399, 0, 0, 0, 0, 0, 'Swift Mountain Horse SAI Script'),
```

Comment:
EN:
the guides of lines 1 and 2 from sai to npc 36741 are wrong, they do not exist in this db.
ES:
las guids de las lineas 1 y 2 del sai al npc 36741 estan erroneas no existen en esta db.